docs/faq/integration.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+sidebar_label: Integration
+---
+
+# Venom Blockchain FAQ: Integration with Venom Blockchain
+
+<details>
+<summary>
+How to connect Venom Wallet to my dApp?
+</summary>
+
+The easiest way to connect your dApp to Venom Wallet is to use [Venom Connect](https://www.npmjs.com/package/venom-connect). It is a library that allows you to connect to Venom Wallet (both mobile and browser extension) and interact with it. This library provides you with a handy interface for building connect popup for our venom wallet and then gives us an interface for working with the venom network.
+
+Check out [this](../build/development-guides/how-to-create-your-own-fungible-tip-3-token/venom-in-action/extend-our-tokensale-with-frontend.md#connecting-venom-wallet-to-your-app) paragraph of the frontend guide, that explains how to use venom-connect in your project. Moreover, you can check the final source code of this guide [here](https://github.com/venom-blockchain/guides/tree/master/tokensale-frontend).
+
+You can read about all configuration options in venom-connect official [repository](https://github.com/web3sp/venom-connect). Also, it has an [example](https://github.com/web3sp/venom-connect/tree/main/examples/react).
+</details>
+
+<details>
+<summary>
+What if I need very specific logic for connecting the Wallet to my dApp?
+</summary>
+
+So, in this case, you can use the library that venom-connect has been built on [inpage-provider](https://github.com/broxus/everscale-inpage-provider) and [standalone-client](https://github.com/broxus/everscale-standalone-client) - basic libraries for interaction with the venom network, so you can build your system for wallet connection. Check the documentation for these libraries in its repositories for more information.
```

Comment:
In this scenario you can use the library that venom-connect has been built on [inpage-provider](https://github.com/broxus/everscale-inpage-provider) and [standalone-client](https://github.com/broxus/everscale-standalone-client) - basic libraries for interaction with the venom network. These will allow you to build your system for wallet connection. Check the documentation for these libraries in their repositories for more information.